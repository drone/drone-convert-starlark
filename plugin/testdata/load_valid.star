# Relative load.
load("@test//:steps_extension.star", "example_step")

# Absolute load.
load("@test//subpackage:pipelines_extension.star", "example_pipeline")

def main(ctx):
    return example_pipeline("sample", steps = example_step())
