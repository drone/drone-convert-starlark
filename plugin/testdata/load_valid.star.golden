{"kind": "pipeline", "type": "docker", "name": "sample", "steps": {"name": "Example step", "image": "plugins/some-test-plugin"}}